### ISSUE TITLE
svg export: axes are not displayed when setting the projection dir

### ISSUE THREAD

**Issue by roipoussiere** [CadQuery/cadquery#731]
Here is a code that render a simple cube in a svg file:

```py
import cadquery.cq
from cadquery.cq import Workplane as WP

figure = WP('XY').box(3, 3, 3)
cq.exporters.export(figure, 'figure.svg')
```

![image](https://user-images.githubusercontent.com/1665542/115020786-58c04280-9ebb-11eb-9dad-4abcda5badff.png)

When I update the projection dir, the axes disappear:

```py
cq.exporters.export(figure, 'figure.svg', opt = {
    'projectionDir': (0.3, -0.2, 0.1)
})
```

![image](https://user-images.githubusercontent.com/1665542/115020935-9624d000-9ebb-11eb-8998-0c47e476b55b.png)

It's maybe related to #599.

**Comment by jmwright** [comment]
Correct on it being related to #599 because the axes are hidden on purpose when setting the projection dir because they're not dynamic. Otherwise the axes indicator would not match the projection direction, causing confusion.

**Comment by jmwright** [comment]
BTW, this behavior is documented in the `showAxes` bullet point in the docs: https://cadquery.readthedocs.io/en/latest/importexport.html#exporting-svg

**Comment by roipoussiere** [comment]
Ok so I close this since it will be fixed in #599.
